llvm/utils/lit/tests/xunit-output.py

```diff
@@ -10,17 +10,17 @@
 # CHECK:      <?xml version="1.0" encoding="UTF-8"?>
 # CHECK-NEXT: <testsuites time="{{[0-9.]+}}">
 # CHECK-NEXT: <testsuite name="test-data" tests="5" failures="1" skipped="3" time="{{[0-9.]+}}">
```

Comment:
If all the times for individual test cases look like `[0-9]+\.[0-9]+`, then perhaps the summary for the test suite and for all the test suites should follow the same pattern? There's no reason we expect those to have multiple decimal points.

(But perhaps having _no_ decimal point is also a possibility? In which case we should allow for that everywhere too.)

Reply:
Done.